packages/core/test/tag-input/tagInputTests.tsx

```diff
@@ -148,13 +148,14 @@ describe("<TagInput>", () => {
             wrapper.setProps({ inputProps: { value: NEW_VALUE } });
             wrapper.find("input").simulate("change", { currentTarget: { value: NEW_VALUE } });
             wrapper.simulate("blur");
+
             // Need setTimeout here to wait for focus to change after blur event
             setTimeout(() => {
                 assert.isTrue(onAdd.calledOnce);
                 assert.deepEqual(onAdd.args[0][0], [NEW_VALUE]);
                 assert.equal(onAdd.args[0][1], "blur");
                 done();
-            });
+            }, 50);
```

Comment:
same here, this seems a little suspect... I feel like there's a semantic difference between setTimeout without a timeout value ("just wait until the next event loop") vs. using an explicit timeout

Reply:
see above